### DISCUSSION TITLE
Roland D20 implementation

### DISCUSSION THREAD

**Discussion by anunseenruler** [keirf/flashfloppy#629] [Q&A]
Hi, new here. The drive on my Roland D20 died and I was directed here by a fellow Roland user who thought flash floppy with a Gotek might work.

my understanding is that this wasn’t previously possible as the drive is proprietary but I am wondering if anyone here has heard otherwise.

Thank you in advance!

**Comment by keirf** [comment]
I think it doesn't use a standard ribbon connector. Perhaps it could work if someone works out which wire goes where on the non-standard drive interface.

**Comment by chrispoacher** [comment]
I have details from the service
Manual but it looks more like an old apple protocol as they dont seem to match up with standard fdc lines. I did try before but to no avail.
